src/config/__tests__/farms.test.ts

```diff
@@ -12,14 +14,31 @@ const farmsToTest: [number, SerializedFarm, number][] = farms56
 
 const farms1ToTest: [number, SerializedFarm, number][] = farms1.slice(0, 10).map((farm) => [farm.pid, farm, 1])
 
+let logged = false
+export const getFarmConfig = async (chainId: ChainId) => {
+  try {
+    return (await import(`../../../packages/farms/constants/${chainId}`)).default.filter(
+      (f: SerializedFarmConfig) => f.pid !== null,
+    ) as SerializedFarmConfig[]
+  } catch (error) {
+    if (!logged) {
+      console.error('Cannot get farm config', error, chainId)
+      logged = true
+    }
+    return []
+  }
+}
+
 describe('Config farms', () => {
-  it.each([...farmsToTest, ...farms1ToTest])('Farm #%d has an unique pid', (pid) => {
-    const duplicates = farms56.filter((f) => pid === f.pid)
+  it.each([...farmsToTest, ...farms1ToTest])('Farm #%d has an unique pid', async (pid, farm, chainId) => {
+    const farmConfig = await getFarmConfig(chainId)
+    const duplicates = farmConfig.filter((f) => pid === f.pid)
```

Comment:
we should test unique pid between all chains no?

Reply:
yeah pid should check bsc chain.
Only lpAddress need check in each chain